drake/systems/plants/collision/ResultCollector.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DRAKE_SYSTEMS_PLANTS_COLLISION_RESULTCOLLECTOR_H_
+#define DRAKE_SYSTEMS_PLANTS_COLLISION_RESULTCOLLECTOR_H_
+
+#include "drake/systems/plants/collision/DrakeCollision.h"
+
+namespace DrakeCollision {
+Eigen::Vector3d toVector3d(const Eigen::Vector3d& vec);
+
+class ResultCollector {
+ public:
+  virtual ~ResultCollector() {}
+
+  virtual void addPointPairResult(const PointPair& result);
```

Comment:
I guess it mostly depends on whether the FCL branch's wrongly-named methods will be grandfathered in or not.  I'll say let's not open an issue now -- I propose we either fix it or grandfather it as part of the FCL PR, rather than worrying about it here.